docs/en/stock-management/stock.settings.md

```diff
@@ -29,61 +29,53 @@ order more stock.
+- **Default minimum number of months of security stock for depots** (integer, default 2 months)
   The stock supply should provide at least this many months of stock at
   all times for the depot involved.
 
-- **Enable automatic confirmation of purchase orders** (yes/NO)  
+- **Minimum Purchase Delay**
+  Sets the minumum amount of expected delay in months for a supplier to deliver medicines after
+  the purchase order is finalised.
+
+- **Enable automatic confirmation of purchase orders** (yes/NO)
   Enable automatic confirmation of purchase orders when they are created
   without requiring manual confirmation
 
-- **Enable permission to view depots in stock registries** (yes/NO)  
+- **Enable permission to view depots in stock registries** (yes/NO)
   Limit the consultation of stock registries to users according to their
   depots
 
-- **Enable automatic stock accounting** (YES/no)  
+- **Enable automatic stock accounting** (YES/no)
   Enabling this feature will write stock movement transactions into the
   posting journal in automatically, in real time. It requires all inventory
   accounts to be correctly configured.
 
-- **Enable automatic crediting of supplier on stock entry** (yes/NO)  
+- **Enable automatic crediting of supplier on stock entry** (yes/NO)
   Automatically adds the transactions to credit the supplier for the value of
   stock received in a depot when entering stock from a purchase order. This is
   only triggered if the automatic stock accounting is set as well.
 
-- **Activate the restriction of distribution depots** (yes/NO)  
+- **Activate the restriction of distribution depots** (yes/NO)
   Limits the depots a user can interact with to the depots that the user has
   access to. Thus, if a user cannot distribute stock from a depot, they also
   cannot transfer stock to that depot.
 
-- **Algorithm for calculating Average Monthly Consumption (AMC)**.  
+- **Algorithm for calculating Average Monthly Consumption (AMC)**.
   Possible choices:
 
-  - **Algorithm 1**: The average monthly consumption is obtained by
+  - **Default Algorithm**: The average monthly consumption is obtained by
     dividing the quantity consumed during the period by the number of
     days having stock for the period, and by multiplying the result
     by 30.5.
 
     ![Algorithm 1](./images/algorithm1.png)
 
-  - **Algorithm 2**: The average consumption is obtained by dividing the
-    quantity consumed during the period by the number of days of
-    consumption for the period, and by multiplying the result by 30.5.
-
-    ![Algorithm 2](./images/algorithm2.png)
-
-  - **Algorithm 3**: The average consumption is obtained by dividing the
-    quantity consumed during the period by the number of days in the
-    period, and by multiplying the result obtained by 30.5.
-
-    ![Algorithm 3](./images/algorithm3.png)
-
   - **Algorithm 4 (MSH)** (default): The average consumption is obtained
```

Comment:
There is still a reference to "Algorithm 4 (MSH)".  It should be changed to "MSH Algorithm".